core-services/prow/02_config/_plugins.yaml

```diff
@@ -2532,6 +2532,8 @@ plugins:
   - trigger
   - verify-owners
   - wip
+  open-cluster-management/api:
+  - approve
```

Comment:
@pmorie it is correct that you want this only for that one repo and not the whole organization>

Reply:
That's correct